tests/js/lintAll-tests.js

```diff
@@ -0,0 +1,90 @@
+"use strict";
+var fluid = require("infusion");
+var jqUnit = require("node-jqunit");
+
+require("../../src/js/lint-all");
+
+fluid.registerNamespace("fluid.test.lintAll");
+
+// TODO: There is never output from one of the tests, and there's no summary.  Discuss with Antranig.
```

Comment:
Which one?

Reply:
"If we run with no custom arguments, the checks should pass."

Reply:
I figured out that the "all checks" promise was never resolved, added a few safety checks for that and found multiple issues with the tests and with the function under test.